theme/base-2016/partials/_footer.twig

```diff
@@ -17,6 +17,8 @@
 
 </footer>
 
-<script src="{{ asset('js/jquery.min.js', 'theme') }}"></script>
+{# Extensions and copypasta gleaned from other sources tend to cause multiple jQuery
+   includes. We only include it here conditionally, if not done so already. #}
+<script>window.jQuery || document.write("<script src='{{ asset('js/jquery.min.js', 'theme') }}'><\/script>");</script>
```

Comment:
So you know. Google is going to start to block document write to inject scripts in chrome for slow connections, non asyn /  defer and a few more here https://developers.google.com/web/updates/2016/08/removing-document-write

Essentially jQuery will never load in chrome 55+ like this